commands/translation_stat/tr_core.py

```diff
@@ -0,0 +1,377 @@
+from github import Github, Repository, GithubObject
+import json
+import os
+from threading import Lock
+from contextlib import nullcontext
+import re
+from datetime import datetime
+
+g: Github = None
+repo: Repository = None
+
+# repo = g.get_repo("placeTW/website")
+main_lang = "en"
```

Comment:
constants should be UPPER_CASE

```suggestion
MAIN_LANG = "en"
```